integrations/RollBar/browser.js

```diff
@@ -0,0 +1,496 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable class-methods-use-this */
+
+import { isObject } from "../../utils/utils";
+import logger from "../../utils/logUtil";
+import { NAME } from "./constants";
+
+class RollBar {
+  constructor(config) {
+    this.name = NAME;
+    this.accessToken = config.accessToken;
+    this.captureUncaughtException = config.captureUncaughtException;
+    this.captureUnhandledRejections = config.captureUnhandledRejections;
+    this.guessUncaughtFrames = config.guessUncaughtFrames;
+    this.codeVersion = config.codeVersion;
+    this.ignoredMessages = config.ignoredMessages;
+    this.environment = config.environment;
+    this.sourceMapEnabled = config.sourceMapEnabled;
+  }
+
+  init() {
+    logger.debug("===in init RollBar===");
+    var _rollbarConfig = {
+      accessToken: this.accessToken,
+      captureUncaught: this.captureUncaughtException,
+      captureUnhandledRejections: this.captureUnhandledRejections,
+      payload: {
+        environment: this.environment,
```

Comment:
what happens if nothing is input. what is the default value

Reply:
unKnon  evironment gets created